packages/integration-tests/api/distribution.js

```diff
@@ -7,50 +7,37 @@ const jwt = require('jsonwebtoken');
 
 const CloudFormation = require('@cumulus/aws-client/CloudFormation');
 const SecretsManager = require('@cumulus/aws-client/SecretsManager');
+const { deprecate } = require('@cumulus/common/util');
 
 const { getEarthdataAccessToken } = require('./EarthdataLogin');
 
 /**
- * Invoke distribution API lambda directly to get a signed S3 URL.
+ * Invoke distribution API lambda directly to get a response payload.
+ *
  * This is used in integration testing so that we use the lambda's IAM
  * role/permissions when accessing resources.
  *
  * @param {string} path
  *   path to file requested.  This is just "/bucket/keytofile"
- * @param {headers} headers - Headers to use for TEA request
+ * @param {string} accessToken
+ *   Access token from OAuth provider or nothing.
  * @returns {string}
  *   signed s3 URL for the requested file.
  */
-async function invokeApiDistributionLambda(
-  path,
-  headers
-) {
+async function invokeApiDistributionLambda(path, accessToken = '') {
```

Comment:
Changing this from a headers object to an accessToken string is also a breaking change.